src/containers/Settings/ExportModal/index.js

```diff
@@ -68,7 +95,28 @@ export class ExportModal extends React.Component {
                   required: true,
                 },
               ],
-            })(<ModalFormInput />)}
+            })(<ModalFormInput type="password" />)}
```

Comment:
Does this also need to be configured with the `onBlur` handler? 

What happens now if the user writes a password in the first box, then enters a wrong password in the second box, and finally updates the password in the first box to correspond with the one in the second? Will it trigger a validation that confirms both input passwords are now identical?

Reply:
Great catch. I am even wondering if you got a biological UI render in order to spot this kind of issue without actually running the codes. 🥇 

I made a fix to the potential issue you pointed out.

Reply:
LOL, I wish :-P